common/config/rush/common-versions.json

```diff
@@ -59,6 +59,6 @@
     "@opentelemetry/api": ["^0.14.0"],
     // Monitor: Allow node 10 types until Timeout / Timer conflict is resolved in OpenTelemetry repo
     // TODO: remove when released https://github.com/open-telemetry/opentelemetry-js/pull/1352
-    "@types/node": ["^10.0.0"]
+    "@types/node": ["^10.0.0", "^10.12.0"]
```

Comment:
you will need to update the comment above as to when we can remove the entry for 10.12.0

Reply:
It should not be removed. I added a comment to explain why it is needed.